eventing/sources/Sources.md

```diff
@@ -0,0 +1,42 @@
+# Knative Event Sources
+
+This is a non-exhaustive list of Event sources for Knative. Inclusion in this list **does not**
+imply an endorsement, nor does it imply any level of support.
+
+
+## Sources
+
+These are sources that are installed as `CRD`s.
+
+Name | Status | Support | Description
```

Comment:
Should we include authorship here?

Reply:
I think that would be nice, but in practice what would it mean for some of our existing ones? Knative authors? I think the URL should point to something that should have that context I think?

Reply:
I agree with @vaikas-google, I don't know how to describe authorship in a better way than who owns the linked repo. Eventually we can use a distinct table for Knative supported sources.